src/sourmash/lca/command_summarize.py

```diff
@@ -181,7 +181,7 @@ def summarize_main(args):
     inp_files = args.query
 
     if args.query_from_file:
-        more_files = sourmash_args.load_file_list_of_signatures(args.query_from_file)
+        more_files = sourmash_args.load_pathlist_from_file(args.query_from_file)
```

Comment:
please create a new issue for testing `--query-from-file` for `sourmash lca summarize` - thanks!

Reply:
(because our current tests don't execute this, apparently!)